.gitignore

```diff
@@ -21,9 +21,10 @@ tool/deployments/*/data
 
 # Go CMD's build output
 go/cmd/berty/berty
-go/cmd/bertydoctor/bertydoctor
-go/cmd/rdvp/rdvp
+go/cmd/berty-doctor/berty-doctor
+go/cmd/berty-integration/berty-integration
 go/cmd/betabot/betabot
+go/cmd/rdvp/rdvp
 betabot.store
```

Comment:
Order alphabeticaly except for `betabot.store` because it's not the main interest.